app/services/burn_up_calculator.rb

```diff
@@ -6,24 +6,28 @@ def call(project)
 
     calculate_points
 
-    @group_by_day
+    @report_data
   end
 
   private
 
   attr_reader :service_full
 
   def initialize_group_by_day
-    @group_by_day = [
-      { name: 'today', data: { Date.current => service_full.group_by_day[Date.current] } },
-      { name: 'real',  data: service_full.group_by_day },
-      { name: 'ideal', data: service_full.group_by_day.dup }
-    ]
+    @report_data = {
+      group_by_day: [
+        { name: 'today', data: { Date.current => service_full.group_by_day[Date.current] } },
+        { name: 'real',  data: service_full.group_by_day },
+        { name: 'ideal', data: service_full.group_by_day.dup }
+      ],
+      total_backlog_points: total_backlog_points
+    }
   end
 
   def total_backlog_points
     stories = service_full.instance_variable_get('@stories')
-    stories.map(&:estimate).compact.sum
+    backlog_stories = stories.select { |story| !story.accepted? }
+    backlog_stories.map(&:estimate).compact.sum
```

Comment:
You could do this in one line:
`stories.reject(&:accepted).map(&:estimate).compact.sum`

Reply:
good call, thanks!